script/search/1.0/search.php

```diff
@@ -36,7 +36,11 @@
     fail('Query string must be set');
   }
 
-  $q = $_REQUEST['q'];
+  $q = trim($_REQUEST['q']);
+
+  if($q == '') {
```

Comment:
Do we allow `q=*`?

Reply:
Yes, we do for now. That returns 45kb result set and is much quicker to run.

Reply:
`q=%` on the other hand has the same problem as `q=`. But one step at a time huh?